curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-pyramid-generator/6610c105bbdacc114d6cdc44.md

```diff
@@ -0,0 +1,63 @@
+---
+id: 6610c105bbdacc114d6cdc44
+title: Step 43
+challengeType: 0
+dashedName: step-43
+---
+
+# --description--
+
+In order to use a function, you need to call it. A <dfn>function call</dfn> tells your application to run the code from the function wherever you choose to call it. The syntax for a function call is the function name followed by parentheses. For example, this code defines and calls a `test` function.
+
+```js
+function test() {
+
+}
+test();
+```
+
+Call your `padRow` function.
+
+# --hints--
+
+You should call the `padRow` function.
+
+```js
+assert.lengthOf(code.match(/padRow\(\)/g), 2);
```

Comment:
I am able to pass by placing the function call inside the padRow function like this. So we will probably need an extra test to account for that
<img width="702" alt="Screenshot 2024-04-09 at 3 16 30 PM" src="https://github.com/freeCodeCamp/freeCodeCamp/assets/67210629/7633cd45-4f21-4af7-8f01-ef3be0a49d8d">